docs/js/badge.js

```diff
@@ -0,0 +1,16 @@
+/**Badge JS**/
+
+const badgeBtn = document.querySelector('#badgeBtn')
+const hideBadge = document.querySelector('#hideBadge')

```

Comment:
You don't need to explicitly select the hideBadge. As you are applying the eventListener to the badgeBtn, you can select the badge associated with that button using the command `badgeBtn.childNodes[1]`. Otherwise problem would arise when you have multiple such buttons then you will have to select all the buttons and badges

Reply:
Okay, will change it.